### PR TITLE
Add crow enemy and alert for player to game scene

### PR DESCRIPTION
**Pull request by DevonHenderson**
### Description:
This pull request adds the Crow and Alert features to the main scene. The changes present in this PR are:
 - Updates the existing player prefab to include the alert
 - Updates Player prefab tag to 'Player' (for collision checks)
 - CrowManager script checks scene for reference to Player(clone) alert object
 - Collision added to player prefab to trigger the CrowCollision behaviour
 - Updates Alert prefab tag to 'Alert' (for finding GameOb in scene)

### To setup in any future scenes:
 - Copy `Managers/CrowManager` gameobject from existing scene into new scene

### CODE REVIEW

File: Assets/Scripts/Crow/CrowManager.cs [hunk 1]
```diff
@@ -94,6 +101,14 @@ private IEnumerator MoveCrow(GameObject crow)
     /// <returns>WaitForSeconds before hiding gameobject again</returns>
     private IEnumerator ShowAlert()
     {
+        //Initial reference to alert gameobject
+        if (!alertFound)
```
Comment (LiamMcKenzie) [inline Assets/Scripts/Crow/CrowManager.cs:105]: instead of having a variable for if the alert is found, you could just check if playerAlert is not null

**Review by LiamMcKenzie — APPROVED** [review]
This feature looks good to merge into staging. 

#15 I thought there might be potential conflicts with this branch, but it has a separate scene so there should be no issue

File: Assets/Scripts/Crow/CrowManager.cs (hunk 1 above)
Comment (DevonHenderson) [reply Assets/Scripts/Crow/CrowManager.cs]: good point, have changed this to just check for null instead and removed the bool entirely

**Review by JohnathanGlasgow — APPROVED** [review]
This is working mostly as expected.
I don't think the crow should always spawn at the beginning, so we'll just use a random delay in GameManager before calling SpawnCrow I guess.
The alert currently needs to assigned and deactivated earlier, see comments.
This is fine otherwise.

**Pull request merged**